openmmtools/states.py

```diff
@@ -2066,25 +2066,26 @@ def __getstate__(self):
         """Return a dictionary representation of the state."""
         # Create original ThermodynamicState to serialize.
         thermodynamic_state = object.__new__(self.__class__.__bases__[1])
-        thermodynamic_state.__dict__ = copy.deepcopy(self.__dict__)
+        thermodynamic_state.__dict__ = self.__dict__
```

Comment:
Is there a reason this is not a deep copy any more? Do you want it pointing right at the same object?